docs/team/charliemoweng.md

```diff
@@ -0,0 +1,87 @@
+      relevant students under them. This task does exactly that.
+    * Highlights: This command will automatically add tasks to all students under the module, relieving the pain of 
+      assigning all tasks to each individual student from the users.
+
+
+* **New Feature**: Added in the feature to mark a task under a student as done. (Pull request [#129](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/129))
+    * What it does: When users provide the module, the student, and the task, this command will mark the specific 
+      task under that module and student as done, which means that the student has completed this task.
+    * Justification: TAB aims to make TAs' life easier (thus the name) and by enabling them to mark a task under a 
+      student as done, TAB makes it very straightforward for TAs to keep track of the progress of each student under 
+      them. It takes just a look and some scrolling to know who has done the task and who has not.
+
+
+* **New Feature**: Added in the feature to mark a task under a student as undone (not done yet). (Pull request [#129](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/129))
+    * What it does: It does just the opposite of what the previous feature does - it marks a task under a student as 
+      undone, which means that the student has not completed the task yet.
+    * Justification: We all make mistakes. Students do, TA do, sometimes even professors do. This feature provideds 
+      a way to right the wrong. If a task has been accidentally marked as done but in fact the student has not done 
+      it yet, this feature allows TAs to revert it back to default (tasks are set as undone by default).
+
+
+* **Enhancements to existing features**:
+    * Added in checks for duplicate Telegram handles and emails when adding or editing a student. So that no two 
+      students under the same module can have the same Telegram handle or email address. (Pull request [#227](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/227))
+    * Fixed the bug that will cause an exception to be thrown when a student is added to a module. (Pull request [#111](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/111))
+    * Wrote tests for Module, Task, and the feature to mark a task as done or undone to increase coverage.
+      (Pull requests [#150](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/150), [#219](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/219), [#224](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/224))
+    * Modified and add on some tests written by others to reflect updates in logic. It was not easy to try to better 
+      the test cases written by others, especially when you are not the one who wrote the logic. I traced the code multiple 
+      times to make sure the test cases that I modified or added will work and improve coverage.
+
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/#breakdown=true&search=charliemoweng)
```

Comment:
I think this should be a longer link? From after clicking the `</>` icon near your name, then copy that URL

Reply:
ok